### PR TITLE
changes in generate all results

### PR DESCRIPTION
**Pull request by jorgepz**

### CODE REVIEW

File: generate_all_examples_results.sh [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 # Run all examples
 #
-examples=("3")
-#
-for i in "${examples[@]}"; do cd examples/Example_${i} && nohup bash reproduce_Example_${i}.sh && cd .. & done
+for i in {1..4}
+do
+  echo "generating Example${i}"
+  cd examples/Example_${i}
+  ./reproduce_Example_${i}.sh
```
Comment (mvanzulli) [inline generate_all_examples_results.sh]:
```suggestion
  nohup bash ./reproduce_Example_${i}.sh
```

**Review by mvanzulli — APPROVED** [review]

**Comment by jorgepz** [comment]
image

**Comment by mvanzulli** [comment]
> image

Now might be solved

**Comment by jorgepz** [comment]
pecable. grazzie mile

**Pull request merged**